recipes/continuable/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+        # Exceptions are disabled and `std::error_condition` is used as error_type. See tutorial-chaining-continuables-fail for details.
+        "with_no_exceptions": [True, False],
+        # Exceptions are disabled and the type defined by `CONTINUABLE_WITH_CUSTOM_ERROR_TYPE` is used as error_type.
+        # See tutorial-chaining-continuables-fail for details.
+        "with_custom_error_type": [True, False],
+        # Allows unhandled exceptions in asynchronous call hierarchies. See tutorial-chaining-continuables-fail for details.
+        "with_unhandled_exceptions": [True, False],
+        # Allows to customize the final callback which can be used to implement custom unhandled asynchronous exception handlers.
+        "with_custom_final_callback": [True, False],
+        # Don't decorate the used type erasure, which is done to keep type names minimal for better error messages in debug builds.
+        "with_immediate_types": [True, False],
+    }
+    default_options = {
+        "with_no_exceptions": False,
+        "with_custom_error_type": False,
+        "with_unhandled_exceptions": False,
+        "with_custom_final_callback": False,
+        "with_immediate_types": False
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        minimal_cpp_standard = "14"
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, minimal_cpp_standard)
+        minimal_version = {
+            "gcc": "5",
+            "clang": "3.4",
+            "apple-clang": "10",
+            "Visual Studio": "14"
+        }
+        compiler = str(self.settings.compiler)
+        if compiler not in minimal_version:
+            self.output.warn(
+                "%s recipe lacks information about the %s compiler standard version support" % (self.name, compiler))
+            self.output.warn(
+                "%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
+            return
+        version = tools.Version(self.settings.compiler.version)
+        if version < minimal_version[compiler]:
+            raise ConanInvalidConfiguration("%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "continuable-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE.txt", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*", dst=os.path.join("include", "continuable"), src=os.path.join(self._source_subfolder, "include", "continuable"))
+
+    def package_info(self):
+        if self.settings.os == "Linux":
+            self.cpp_info.system_libs.append("pthread")
+        if self.options.with_no_exceptions:
+            self.cpp_info.defines.append('CONTINUABLE_WITH_NO_EXCEPTIONS')
```

Comment:
When going over this recipe again, can you please change all occurrences of single quote `'` to double quote `"`?
This makes the recipe more uniform.